.github/workflows/build-git-installers.yml

```diff
@@ -641,38 +641,42 @@ jobs:
     steps:
       - name: Clone repository
         uses: actions/checkout@v2
+        with:
+          path: 'git'
       - name: Download unsigned packages
         uses: actions/download-artifact@v2
         with:
           name: deb-package-unsigned
-          path: ${{ env.ARTIFACTS_DIR }}/unsigned
+          path: unsigned
       - uses: azure/login@v1
         with:
           creds: ${{ secrets.AZURE_CREDENTIALS }}
-      - name: Download ESRP client
-        run: |
-          az storage blob download --subscription "${{ secrets.AZURE_SUBSCRIPTION }}" --account-name msftgitesrp -c microsoft-esrp-client -n microsoft.esrpclient.1.2.76.nupkg -f esrp.zip
-          Expand-Archive -Path esrp.zip -DestinationPath .\esrp
-      - name: Install ESRP certificates
+      - name: Set up ESRP client
```

Comment:
I appreciate that you found ways to combine the ESRP work across the two platforms.